addons/compositions/XEH_preInit.sqf

```diff
@@ -8,4 +8,25 @@ PREP_RECOMPILE_END;
 
 GVAR(randomize) = false;
 
+private _compositions = GET_COMPOSITIONS;
+
+if (typeName _compositions isEqualTo "ARRAY") then {
```

Comment:
`isEqualType`.

Reply:
This was done on purpose as a hash is also seen as an array by isEqualType, unless that has changed in a recent update.

Reply:
Looks to be working as expected now:

```sqf
createHashMap isEqualType [] // -> false
```